addon/mixins/model-records/resource-list-controller.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember'
+
+const {
+  Mixin
+} = Ember
+
+export default Mixin.create({
+  queryParams: ['filter[keyword]', 'page', 'size'],
+  filter: null,
+  page: 1,
+  size: 30,
+  actions: {
+    transitionToRoute(...args) {
```

Comment:
mentioned this above, but better to likely pass the actions data up and handle it here.  Or maybe just rename the action to updateQueryParams